chaos_kitten/cli.py

```
@@ -171,6 +168,21 @@ def scan(
         min=1,
         max=5,
     ),
+    resume: bool = typer.Option(
```

Ruff (0.15.2)

[warning] Unused function argument: `resume`

(ARG001)